src/System.Diagnostics.StackTrace/src/System.Diagnostics.StackTrace.csproj

```diff
@@ -27,6 +32,18 @@
   <ItemGroup>
     <Compile Include="System\Diagnostics\StackFrameExtensions.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == ''">  
+    <Compile Include="System\Diagnostics\StackTraceSymbols.CoreCLR.cs" />  
+    <ProjectReference Include="..\..\System.Collections\src\System.Collections.csproj" />  
+    <ProjectReference Include="..\..\System.Threading.Overlapped\src\System.Threading.Overlapped.csproj" />  
```

Comment:
If you build locally with build.cmd you'll see the failure.

System.Threading.Overlapped only builds with <OSGroup>Windows_NT</OSGroup>.  You need to add that metadata here to the project reference so that you don't build that project with different global properties.  It's more about not introducing a race condition in the build than actually building for Windows.  You may not even need the overlapped reference at all, so if you can remove it that'd be better.